src/DropInViewer.js

```diff
@@ -119,6 +131,20 @@ export class DropInViewer extends THREE.Group {
     );
   }
 
+  /*
+   * Uses the raycaster to traverse the different splats and checks for collisions.
+   *
+   * @param {object} camera is the perspective camera used to render
+   * @param {object} position is the normalized position relative to the screen.
+   * @param {object} screenSize
+   * @returns {object|null} the first splat that collides with the ray.
+   */
+  getSplatPosition = (function() {
+    return function(rendererSize, camera, position) {
+      return this.viewer.getSplatPosition(rendererSize, camera, position);
+    };
+  })();
```

Comment:
why do we need the [SIAF](https://blog.mgechev.com/2012/08/29/self-invoking-functions-in-javascript-or-immediately-invoked-function-expression/) here? Am I missing something?

```suggestion
  getSplatPosition(rendererSize, camera, position) {
    return this.viewer.getSplatPosition(rendererSize, camera, position);
  };
```

or

```suggestion
  getSplatPosition = (rendererSize, camera, position) => {
    return this.viewer.getSplatPosition(rendererSize, camera, position);
  };
```

Reply:
Removed